DependencyInjection/Configuration.php

```diff
@@ -108,6 +246,12 @@ public function getConfigTreeBuilder()
                                 // boolean types
```

Comment:
@tobias-93 duplicated code, let's replace `->defaultValue(array( ...` with `->defaultValue($this->defaultFormTypes['doctrine_orm'])` and similar change to all below